### DISCUSSION TITLE
Segmentation fault

### DISCUSSION THREAD

**Discussion by RorroPoblete** [IIC2133-PUC/2024-1#58] [Ayuda]
Hola, el test 2.4-inc me arroja segmentation faul y el 2.4 no por lo que no sé como arreglarlo.
Además el test 3.2 parte trabajando correctamente y en un momento empieza a tener diferencias.

**Comment by CataAlamos** [comment]
Hola! Segmentation fault es un error muy amplio, puedes mandar tu output de valgrind?
